protobuf/protobuf.go

```diff
@@ -0,0 +1,150 @@
+package protobuf
+
+import (
+	"fmt"
+	"path/filepath"
+)
+
+// Package represents an unique .proto file with its own package definition.
+type Package struct {
+	Name     string
+	Path     string
+	Imports  []string
```

Comment:
Maybe this should be `Imports []*Package`. This way we could access all information if needed.